app/services/editor-commands/editor-commands.ts

```diff
@@ -63,7 +63,7 @@ export class EditorCommandsService extends StatefulService<IEditorCommandsServic
   executeCommand<TCommand extends keyof typeof COMMANDS>(
     commandType: TCommand,
     ...commandArgs: ConstructorParameters<(typeof COMMANDS)[TCommand]>
-  ) {
+  ): ReturnType<InstanceType<(typeof COMMANDS)[TCommand]>['execute']> {
```

Comment:
Wow, TS magic

Reply:
I know right haha.  Took a while to find the secret TS formula.